src/util/form.helper.js

```diff
@@ -238,15 +245,15 @@ export const groupFields = {
     label: "Wie soll deine Gruppe heißen?",
     placeholder: "Name",
     hint: "Bitte gib den Gruppen Namen ein.",
-    validate: [required, (value) => maxLength(value, 70)],
+    validate: [minCharRequired, (value) => maxLength(value, 70)],
```

Comment:
keep required and simply add minCharRequired to array
```suggestion
    validate: [required, (value) => minCharRequired(value, 3), (value) => maxLength(value, 70)],
```